tuf/ngclient/_internal/requests_fetcher.py

```diff
@@ -96,12 +98,16 @@ def _chunks(self, response: "requests.Response") -> Iterator[bytes]:
         download."""
 
         try:
-            for data in response.iter_content(self.chunk_size):
+            while True:
```

Comment:
Alternatively, we could perhaps replace this `while` loop by the following:

```python
yield from response.raw.stream(amt=self.chunk_size, decode_content=False)
```

See `urrlib3` source: https://github.com/urllib3/urllib3/blob/1f2f6a5817fd420d65e280e671d2b579791aef09/src/urllib3/response.py#L748

Reply:
@lukpueh @joshuagl @MVrachev What do you think about this?